src/esp/assets/ResourceManager.cpp

```diff
@@ -2258,8 +2257,17 @@ void ResourceManager::createConvexHullDecomposition(
     const VHACDParameters& params,
     const bool saveChdToObj) {
   if (resourceDict_.count(filename) == 0) {
+    // retrieve existing, or create new, object attributes corresponding to
+    // passed filename
+    auto objAttributes =
+        getObjectAttributesManager()->getObjectCopyByHandle(filename);
+    if (objAttributes == nullptr) {
+      objAttributes =
+          getObjectAttributesManager()->createObject(filename, false);
+    }
+
```

Comment:
Note this change - the object being decomposed needs to have an attributes, at least to pass to loadObjectMeshDataFromFile, since this function is going to process shader_type directives from configs.  If no attributes for the source object exists, one is made, but not registered.